.taskcluster.yml

```diff
@@ -10,302 +10,305 @@ tasks:
+                  else:
+                      $if: 'tasks_for == "cron"'
+                      then: '${ownTaskId}'
+              pullRequestAction:
+                  $if: 'tasks_for == "github-pull-request"'
+                  then: ${event.action}
+                  else: 'UNDEFINED'
+              releaseAction:
+                  $if: 'tasks_for == "github-release"'
+                  then: ${event.action}
+                  else: 'UNDEFINED'
+          in:
+              $if: >
+                tasks_for in ["action", "cron"]
+                || (tasks_for == "github-pull-request" && pullRequestAction in ["opened", "reopened", "synchronize"])
+                || (tasks_for == "github-push" && head_branch[:10] != "refs/tags/" && head_branch != "staging.tmp" && head_branch != "trying.tmp")
+                || (tasks_for == "github-release" && releaseAction == "published")
+              then:
+                  $let:
+                      level:
+                          $if: 'tasks_for in ["github-push", "github-release", "action", "cron"] && repoUrl == "https://github.com/mozilla-mobile/reference-browser"'
+                          then: '3'
+                          else: '1'
 
-                  ownTaskId:
-                      $if: '"github" in tasks_for'
-                      then: {$eval: as_slugid("decision_task")}
-                      else:
-                          $if: 'tasks_for == "cron"'
-                          then: '${ownTaskId}'
-
-                  pullRequestAction:
-                      $if: 'tasks_for == "github-pull-request"'
-                      then: ${event.action}
-                      else: 'UNDEFINED'
-
-                  releaseAction:
-                      $if: 'tasks_for == "github-release"'
-                      then: ${event.action}
-                      else: 'UNDEFINED'
-              in:
-                  $if: >
-                    tasks_for in ["action", "cron"]
-                    || (tasks_for == "github-pull-request" && pullRequestAction in ["opened", "reopened", "synchronize"])
-                    || (tasks_for == "github-push" && head_branch[:10] != "refs/tags/" && head_branch != "staging.tmp" && head_branch != "trying.tmp")
-                    || (tasks_for == "github-release" && releaseAction == "published")
-                  then:
-                      $let:
-                          level:
-                              $if: 'tasks_for in ["github-push", "github-release", "action", "cron"] && repoUrl == "https://github.com/mozilla-mobile/reference-browser"'
-                              then: '3'
-                              else: '1'
+                      short_head_branch:
+                          $if: 'head_branch[:11] == "refs/heads/"'
+                          then: {$eval: 'head_branch[11:]'}
+                  in:
+                      $mergeDeep:
+                          - $if: 'tasks_for != "action"'
```

Comment:
This is the actual fix to the aforementioned error. It exists in fenix. The following changes are just about shifting everything to the right.